### DISCUSSION TITLE
EJBCA cmp + openssl runtime produces error

### DISCUSSION THREAD

**Discussion by XYQ202** [Keyfactor/ejbca-ce#56] [General]
hello team，
      For the installation of EJBCA, refer to the quick script installation in the documentation of the official website, which has been successfully installed on ubuntu16, and can enter the CA UI, which can work normally.
Refer to the CMP Operations Guide in the documentation on your official website to try the CMP function. First, a CMP alias "opensslca" is generated through the interface, then the cmp alias is configured, and then the given command is passed in ubuntu, but no A successful run produces a certificate signed by ManagementCA.
     1. openssl genrsa 2048 > key1.pem.
          Successfully generated the RSA key of key1.pem
     2. openssl cmp -cmd ir -server localhost:8080 -path ejbca/publicweb/cmp/opensslra -srvcert ManagementCA.cacert.pem -ref NewUser -secret pass:password -certout clcert1.pem -newkey key1.pem -subject "/CN=NewUser/O=My Organization/C=SE".
          After running, there are some errors, can you help me?
          The print after running is as follows:
                 
![image](https://user-images.githubusercontent.com/97773232/177118343-d47d4210-1a98-4a5a-8071-d7b512d39986.png)

**Comment by primetomas** [comment]
The error says "no such file: ManagementCA.pem"
Just make sure you have that file in the filesystem, and use the full path to point to it.

**Reply by XYQ202** [reply]
> The error says "no such file: ManagementCA.pem" Just make sure you have that file in the filesystem, and use the full path to point to it.

Thanks.It has been solved, you can register to obtain a certificate through openssl cmp normally

However, when I installed the official website and executed the renewal certificate (KUR) through openssl cmp, it has been unsuccessful. The error is: 
               CMP Error: missing protection

**Comment by primetomas** [comment]
Great. I see you created #57 for the follow up issue.
